TF_Tooling_Web/src/Plugins/InputActionPlugin.ts

```diff
@@ -0,0 +1,40 @@
+import { TouchFreeInputAction } from "../TouchFreeToolingTypes";
+
+export class InputActionPlugin extends EventTarget{
```

Comment:
I think we should make [this class `abstract`](https://www.typescriptlang.org/docs/handbook/2/classes.html#abstract-classes-and-members) as it is in the Unity version to prevent people trying to _use_ this plugin